src/OpenLoco/src/Vehicles/OrderManager.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "LabelFrame.h"
+#include "Types.hpp"
+#include <OpenLoco/Engine/World.hpp>
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace OpenLoco::Vehicles
+{
+    struct VehicleHead;
+}
+
+namespace OpenLoco::Vehicles::OrderManager
+{
+    struct NumDisplayFrame
+    {
+        uint32_t orderOffset; // 0x0
+        LabelFrame frame;     // 0x4
+        uint8_t lineNumber;   // 0x24
+    };
+
+    uint8_t* orders();
+    uint32_t& numOrders();
+    void freeOrders(VehicleHead* const head);
+
+    void sub_470795(const uint32_t removeOrderTableOffset, const int16_t sizeOfRemovedOrderTable);
```

Comment:
const for declarations makes no sense, you can add const on the implementation, see https://godbolt.org/z/Yq9h5fsfq also see https://clang.llvm.org/extra/clang-tidy/checks/readability/avoid-const-params-in-decls.html

Reply:
That's the first I'm hearing of this. It was my impression const is generally preferred, though particularly when passing const refs, of course.

Regardless, I'm pretty sure we do this in ample places in the code base. Perhaps we should do a general cleanup of this in a bigger PR then?

Reply:
Fair enough